src/Stripe.net/Services/PaymentMethods/PaymentMethodUpdateOptions.cs

```diff
@@ -0,0 +1,13 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class PaymentMethodUpdateOptions : BaseOptions
+    {
+        [JsonProperty("metadata")]
+        public Dictionary<string, string> Metadata { get; set; }
```

Comment:
I compare with autogen params, and we seem to allow updating card and billing details too..
Maybe we don't want this?

Reply:
Nice catch, forgot to fix that one. cc @ob-stripe Are we okay having a nested option have names such as `PaymentMethodCardCreateOptions` and `PaymentMethodCardUpdateOptions`?

Reply:
I think that's fine. I assume most users rely on Intellisense to discover classes anyway.